surveys/2023-annual-survey/questions.md

```diff
@@ -0,0 +1,769 @@
+
+
+### To what extent is Rust currently being used by your company?
+
+Type: select one
+
+- My company makes non-trivial use of Rust (e.g., used in production or in significant tooling)
+- My company has experimented with Rust or is considering using it
+- My company has not seriously considered Rust for any use [`NEXT`](#approximately-how-many-total-developers-does-your-company-employ)
+- I am unsure whether my company has considered using or currently uses Rust [`NEXT`](#approximately-how-many-total-developers-does-your-company-employ)
+
+> **justification**
+>
+> We want to establish how reliant companies are on Rust.
+
+
+### In what technology domain(s) is Rust used at your company?
+
+If you've previously answered that your company is not actively using Rust, you can leave this question blank.
+
+Type: select all that apply (optional)
+
+- Audio programming
+- Blockchain
+- Cloud computing applications
+- Cloud computing infrastructure or utilities
+- Command Line Interfaces (CLI)
+- Computer graphics
+- Computer games
+- Computer networking
+- Computer security
+- Data science
+- Database implementation
+- Desktop computer application frontend
+- Desktop computer or mobile phone libraries or services
+- Distributed systems
+- Embedded devices (with operating systems)
+- Embedded devices (bare metal)
+- HPC (High-performance [Super]Computing)
+- IoT (Internet of Things)
+- Machine learning or AI
+- Mobile phone application frontend
+- Programming languages and related tools (including compilers, IDEs, standard libraries, etc.)
+- Robotics
+- Scientific and/or numerical computing
+- Server-side or "backend" application
+- Simulation
+- Web application frontend
+- WebAssembly
+- Other (open response)
+
+> **justification**
+>
+> We want to known roughly what technology stacks are being most often used.
+>
+> This can be ambiguous and hard to answer. For example, if you're building an operating
+> system for a mobile phone, is that embedded, mobile, or something else?
+> We want to understand the "shape" of Rust usage, and this question tries to get at that
+> by allowing the respondent to select multiple answers.
+
+### Approximately how many total developers does your company employ?
+
+**Note**: don't worry about being exact here! Go with you gut.
+
+Type: select one (optional)
+
+- Under 10
+- 10-99
+- 100-1,000
+- 1,000-10,000
+- Over 10,000
+
+> This question is not that interesting on its own, but it can be used as a sort of cohort for understanding how answers
+> change depending on the size of the development effort at a company.
+>
+> Previously this question used "employees" instead of "developers". It is more appropriate for us to ask about the amount
+> of developers at a company vs. the amount of people employed in total.
+
+### Is your company planning on hiring Rust developers in the next year?
+
+Type: select one (optional)
+
+- Yes
+- No
```

Comment:
Distinguishing between "our company isn't hiring developers at all" and "our company isn't hiring in Rust" would be very useful.

Reply:
:+1: It's true that these days this is an important distinction.

Reply:
Changed it to this:
```
- Yes
- No (it is planning to hire other developers)
- No (it is not planning to hire any developers)
- I don't know
```

Reply:
Feel free to reopen if the wording doesn't seem right.